frontend/context/PageStateProvider.tsx

```diff
@@ -35,12 +42,26 @@ export const PageStateProvider = ({ children }: PropsWithChildren) => {
     },
     pageState === Pages.Setup || isPageLoadedAndOneMinutePassed
       ? null
-      : LAST_TRANSACTION_SHOW_DELAY,
+      : ONE_MINUTE_INTERVAL,
   );
 
+  const userLogin = useCallback(() => {
```

Comment:
I'd rename it to `setUserLoggedIn` b/c it creates a small confusion that here we do the exact login, but up to you